test/ambuda/test_models.py

```diff
@@ -1,5 +1,11 @@
 import ambuda.database as db
-from ambuda.queries import get_engine, get_session
+from ambuda.queries import get_session
+
+
+def _cleanup(session, *objects):
```

Comment:
I think there's need to cleanup because in our current testing approach, each new `client` test should recreate the database from scratch.

Reply:
I don't think this currently happens. I noticed errors when trying to create a User with the same email etc. as the test above this. I could've made a different user however I wanted to clean up the side effects from the existing tests in case any other contributors run into this issue. We should investigate and address this issue though - each test should be unique.

Reply:
Fair enough -- let's keep it if it helps the tests pass, and let's find a way around it in a follow-up PR.